plugins/opercmds.py

```diff
@@ -5,13 +5,16 @@
 from pylinkirc.log import log
 from pylinkirc.coremods import permissions
 
-@utils.add_cmd
+checkban_parser = utils.IRCParser()
+checkban_parser.add_argument('banmask')
+checkban_parser.add_argument('-c', '--channel', action='store', default=False)
 def checkban(irc, source, args):
-    """<banmask (nick!user@host or user@host)> [<nick or hostmask to check>]
+    """<banmask (nick!user@host or user@host)> [<nick or hostmask to check> -c/--channel #channel] 
```

Comment:
Separate the two fields: `[<nick or hostmask to check>] [-c/--channel #channel]`